src/core_plugins/kbn_vislib_vis_types/public/horizontal_bar.js

```diff
@@ -0,0 +1,152 @@
+import VislibVisTypeVislibVisTypeProvider from 'ui/vislib_vis_type/vislib_vis_type';
+import VisSchemasProvider from 'ui/vis/schemas';
+import pointSeriesTemplate from 'plugins/kbn_vislib_vis_types/editors/point_series.html';
+
+export default function PointSeriesVisType(Private) {
```

Comment:
I'm getting the following console error when trying to create a horizontal bar chart
```
TypeError: visType is not a function
    at new VisConfig (vis_config.js:50)
    at Vis.render (vis.js:122)
    at Vis.resize (vis.js:153)
    at Function.wrapper (index.js:3095)
    at events.js:60
    at processQueue (angular.js:14745)
    at angular.js:14761
    at Scope.$eval (angular.js:15989)
    at Scope.$digest (angular.js:15800)
    at angular.js:16028
```